src/main/java/microsoft/exchange/webservices/data/ChangeCollection.java

```diff
@@ -85,7 +85,7 @@ public int getCount() {
    */
   public TChange getChangeAtIndex(int index) {
     if (index < 0 || index >= this.getCount()) {
-      throw new IndexOutOfBoundsException(Strings.IndexIsOutOfRange);
+      throw new IndexOutOfBoundsException("index is out of range.");
```

Comment:
maybe we should give a more suitable message here since IndexOutOfBoundsException simply indicates that the index is out of range. So this message gives no extra information.